README.md

```diff
@@ -14,94 +14,159 @@ The established standard Nix formatting differs considerably from the original o
 
 For more details, see the [RFC implementation tracking issue](https://github.com/NixOS/nixfmt/issues/153).
 
-## Installation And Usage Instructions
+## Installation
 
-### nixpkgs/NixOS
+> [!NOTE]
+> `nixfmt` can only process one file at a time.
+> Consider using a configuration helper for [formatting a project](#in-a-project).
 
-`nixfmt` was used as the basis for the official Nix formatter with a standardized formatting.
-The new formatting differs considerably from the original one.
-A recent nixfmt version is available as `pkgs.nixfmt-rfc-style` in Nixpkgs.
-The formatting of this version differs considerably from the original nixfmt that was used as the basis for the standardised official formatter, which is also still available as `pkgs.nixfmt-classic` for now, though unmaintained.
+### In the environment
 
-So installing this `nixfmt` is as simple as adding to the system packages:
+#### NixOS
+
+To install `nixfmt` on NixOS for all users, add it to the [`environment.systemPackages`](https://search.nixos.org/options?show=environment.systemPackages) configuration option:
```

Comment:
This is not your fault, but IMO this is doubly weird advice:

- We *just* told people `nixfmt` isn't that useful to use as it can only process one file at a time.
- It implies that people would want to format all nix code on their computer with the same version of `nixfmt`. This sort of bogus assumption doesn't apply when nix gives us good tooling to isolate dependencies to projects.

I would rather remove these two sections (NixOS and Home Manager) in favor of the "Declarative shell environment" example. If folks really want a system (or user) install of `nixfmt-rfc-style`, I'm confident they can figure that out once they know the package name.

Reply:
@fricklerhandwerk and I discussed this, and we decided to leave this here. It *is* weird to install a specific version of nixfmt globally, but it's also necessary for the editor configuration we document below.

We could potentially add more words to explain this, but this is how things are before this PR. I'd rather merge this PR and iterate on this in the future.